end_to_end_tests/golden-record/README.md

```diff
@@ -41,6 +41,25 @@ my_data: MyDataModel = await get_my_data_model.asyncio(client=client)
 response: Response[MyDataModel] = await get_my_data_model.asyncio_detailed(client=client)
 ```
 
+By default when you're calling an https api it will attempt to verify that ssl is working correctly.
+Using certificate verification is highly recommended most of the time and is the default.
+But sometimes you may need to authenticate to a server (especially an internal server) using a custom certificate bundle.
```

Comment:
```suggestion
By default, when you're calling an HTTPS API it will attempt to verify that SSL is working correctly. Using certificate verification is highly recommended most of the time, but sometimes you may need to authenticate to a server (especially an internal server) using a custom certificate bundle.
```